.gitignore

```diff
@@ -25,3 +25,5 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+config/
```

Comment:
so I had written a bunch of config stuff before we talked, I ignored it so I wouldn't accidentally commit it at some point - this will just be until we figure out what we want to do with the config.